hazelcast/src/main/java/com/hazelcast/map/impl/operation/EntryOffloadableSetUnlockOperation.java

```diff
@@ -151,18 +156,27 @@ protected void writeInternal(ObjectDataOutput out) throws IOException {
         UUIDSerializationUtil.writeUUID(out, caller);
         out.writeLong(begin);
         out.writeObject(entryBackupProcessor);
+        // RU_COMPAT_4_0
+        if (out.getVersion().isGreaterOrEqual(Versions.V4_1)) {
+            out.writeLong(newTtl);
+        }
     }
 
     @Override
     protected void readInternal(ObjectDataInput in) throws IOException {
         super.readInternal(in);
         String modificationTypeName = in.readUTF();
-        modificationType = modificationTypeName.equals("") ? null : EntryEventType.valueOf(modificationTypeName);
+        modificationType = modificationTypeName.equals("")
+                ? null : EntryEventType.valueOf(modificationTypeName);
         oldValue = IOUtil.readData(in);
         newValue = IOUtil.readData(in);
         caller = UUIDSerializationUtil.readUUID(in);
         begin = in.readLong();
         entryBackupProcessor = in.readObject();
+        // RU_COMPAT_4_0
+        if (in.getVersion().isGreaterOrEqual(Versions.V4_1)) {
+            newTtl = in.readLong();
```

Comment:
Is the default value of 0 ok during RU?

Reply:
yes it is ok, zero will be translated to infinity.